sql/npc_spawn_points.sql

```diff
@@ -0,0 +1,124 @@
+-- MySQL dump 10.13  Distrib 5.6.13, for Win64 (x86_64)
+--
+-- Host: localhost    Database: dspdb
+-- ------------------------------------------------------
+-- Server version	5.6.13-log
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `npc_spawn_points`
+--
+
+DROP TABLE IF EXISTS `npc_spawn_points`;
```

Comment:
I totally see your point, yet I still got 2 arguments/questions:
- the table structure is the same as the treasure_spawn_points, so if that one get's batched, this one should be the same 
- there are many more "moving/jumping npc's/???", and maybe the HELM locations could be moved into this table aswell. that way most of the npcid would be in a central, easily batchable location

nevertheless, if you guys say move it to lua and do it like HELM, I do so and rewrite it